client/test/e2e/shared/components/bhDateEditor.js

```diff
@@ -15,8 +15,8 @@ module.exports = {
    * @param {Date} date - a date object
    * @param {string} id - a CSS id to select on.
    */
-  set: function set(date, id) {
-
+  set: function set(date, id, elementClick) {
```

Comment:
Could you update the documentation comments above this function?  It is missing a third `@param` declaration.